web/src/overlay_util.ts

```diff
@@ -13,3 +13,26 @@ export function disable_scrolling(): void {
 export function enable_scrolling(): void {
     $("html").css({"overflow-y": "scroll", "--disabled-scrollbar-width": "0px"});
 }
+
+export function trapFocus(overlay: JQuery): void {
```

Comment:
We prefer snake_case function and variable names.

Reply:
Done